app/views/recipes/new.html.erb

```diff
@@ -1 +1,42 @@
-<h1>To Create a new recipe page</h1>
+<div class="container d-flex flex-column align-items-center">
+  <h2>Add new recipe:</h2>
+  <%= form_with model: @recipe do |form| %>
+    <div class="form-group mb-3">
+      <%= form.label :name %> <br>
+      <%= form.text_field :name, class: "form-control" , required: true %>
+    </div>
+
+    <div class="form-group mb-3">
+      <%= form.label :description %> <br>
+      <%= form.text_area :description, class: "form-control", required: true %>
+    </div>
+
+    <div class="form-group mb-3">
+      <%= form.label :cooking_time %> <br>
+      <%= form.number_field :cooking_time, step: 1, class: "form-control", required: true %>
+    </div>
+
+    <div class="form-group mb-3">
+      <%= form.label :preparation_time %> <br>
+      <%= form.number_field :preparation_time, step: 1, class: "form-control", required: true %>
+    </div>
+
+    <div class="form-check form-switch mb-3">
+      <%= form.label :public, class: "form-check-label" %>
+      <%= form.check_box :public, class: "form-check-input" %>
+    </div>
+
+    <div class="form-group mb-3">
+      <%= form.label :foods %><br>
+      <%= form.select :foods,
+        options_from_collection_for_select(@foods, :id, :name),
+        { include_blank: true },
+        multiple: true,
+        size: 5,
+        class: 'form-select'
+      %>
+    </div>
+
```

Comment:
- [ Optional ] I think this `select_option` form element can also be removed because we are not using it.